package.json

```
@@ -16,8 +16,9 @@
     "urql": "1.7.0"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "introspect": "./introspect-schema",
+    "start": "yarn introspect && react-scripts start",
+    "build": "yarn introspect && react-scripts build",
```

```suggestion
    "build": "NODE_ENV=production yarn introspect && react-scripts build",
```

Fixes the issue seen in CI